src/qml/components/ChatMessage.qml

```diff
@@ -297,6 +318,10 @@ Control {
         }
     }
 
+    function btToMb(bytes) {
```

Comment:
Remove once you move logic to new C++ role

Reply:
Ok. This was already removed in my local repo.